scripts/generate_json.py

```diff
@@ -74,12 +74,12 @@ def add_new_rollouts(spec_ids, overwrite):
     modified = False
     for spec in environments:
         if not overwrite and spec.id in rollout_dict:
-            logger.debug("Rollout already exists for {}. Skipping.".format(spec.id))
+            logger.debug(f"Rollout already exists for {spec.id}. Skipping.")
         else:
             modified = update_rollout_dict(spec, rollout_dict) or modified
 
     if modified:
-        logger.info("Writing new rollout file to {}".format(ROLLOUT_FILE))
+        logger.info(f"Writing new rollout file to {ROLLOUT_FILE}")
```

Comment:
Admittedly this isn't a new problem, but one of the things you may want to do here (especially since this will turn into a `pylint` error down the line) is use `%` formatting here instead of f-strings, see [here](https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation) for why, and `pylint` (and flake8 -- not sure?) will throw a warning at you down the line